src/main/java/com/stripe/net/RequestOptions.java

```diff
@@ -267,29 +290,25 @@ public RequestOptionsBuilder clearStripeAccount() {
       return setStripeAccount(null);
     }
 
-    public String getStripeVersionOverride() {
+    /** For internal use only. */
+    public String _getStripeVersionOverride() {
```

Comment:
```suggestion
   String getStripeVersionOverride() {
```

same package as test so doesn't need to be public